Smooch.postman_collection.json

```diff
@@ -3538,6 +3538,35 @@
 						}
 					]
 				},
+				{
+					"name": "Get SDK IDs",
+					"request": {
+						"method": "GET",
+						"header": [
+							{
+								"key": "Authorization",
+								"value": "Bearer {{JWT}}"
+							},
+							{
+								"key": "Content-Type",
+								"value": "application/json"
+							}
+						],
+						"url": {
+							"raw": "{{url}}/{{apiVersion}}/apps/{{appId}}",
+							"host": [
+								"{{url}}"
+							],
+							"path": [
+								"{{apiVersion}}",
+								"apps",
+								"{{appId}}"
+							]
+						},
+						"description": "### For more information\n- API Docs: [https://docs.smooch.io/rest/#get-sdk-ids](https://docs.smooch.io/rest/#get-sdk-ids)"
+					},
+					"response": []
```

Comment:
I see examples below have `response` filled, is that a place for an example of what the `response` is? If so maybe we could add it? 🤔🤷‍♂️

Reply:
How do I do that in postman?

Reply:
Figured it out, fixed I think

Reply:
Glad you did, cuz I would not have known how! 😂